backend/api/auth/views.py

```
@@ -311,36 +304,33 @@ def check_account_auth(request):
-        logger.debug("Password reset token for %s: %s", email, reset_token)
-
-        if SEND_EMAILS:
-            send_mail(
-                subject="Plancake - Reset Password",
-                message=f"Click this link to reset your password:\n{BASE_URL}/reset-password?token={reset_token}\n\nNot you? Nothing to worry about, just ignore this email.",
-                from_email=None,  # Use the default from settings
-                recipient_list=[email],
-                fail_silently=False,
+            PasswordResetCode.objects.update_or_create(
+                user_account=user,
+                defaults={"reset_code": reset_code, "created_at": datetime.now()},
```

Ruff (0.16.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)